daemon/cmd/sysctl_linux_test.go

```diff
@@ -32,6 +32,6 @@ type DaemonPrivilegedSuite struct{}
 var _ = Suite(&DaemonPrivilegedSuite{})
 
 func (s *DaemonPrivilegedSuite) TestInitSysctlParams(c *C) {
```

Comment:
Shouldn't we also change the name of the test? :)

Reply:
Yes! Changing the name now...